src/codegen/general.rs

```diff
@@ -394,22 +394,24 @@ pub fn define_auto_boxed_type(
     )?;
     writeln!(w)?;
     writeln!(w, "\tmatch fn {{")?;
-    writeln!(
-        w,
-        "\t\tcopy => |ptr| {}({}::{}(), ptr as *mut _) as *mut {}::{},",
-        use_glib_type(env, "gobject_ffi::g_boxed_copy"),
-        sys_crate_name,
-        get_type_fn,
-        sys_crate_name,
-        glib_name
-    )?;
-    writeln!(
-        w,
-        "\t\tfree => |ptr| {}({}::{}(), ptr as *mut _),",
-        use_glib_type(env, "gobject_ffi::g_boxed_free"),
-        sys_crate_name,
-        get_type_fn
-    )?;
+    if let Some((ref get_type_fn, _get_type_version)) = get_type_fn {
```

Comment:
This requires a `get_type` function, or otherwise more changes in `glib` are needed

Reply:
For the case of not `BoxedInline` but normal `Boxed`

Reply:
So the get_type_fn here can't be None then?

Reply:
Well, there won't be a copy/free function here otherwise and that won't compile without further changes in glib